src/cogent3/core/alignment.py

```diff
@@ -2955,17 +2965,24 @@ def to_html(
         ]
         return "\n".join(text)
 
-    def to_pretty(self, name_order=None, interleave_len=None):
+    def to_pretty(self, name_order=None, interleave_len=None, wrap=None):
         """returns a string representation of the alignment in pretty print format
 
         Parameters
         ----------
         name_order
             order of names for display.
         interleave_len
-            maximum number of printed bases, defaults to alignment length
-
+            maximum number of printed bases, defaults to alignment length, will be replaced by wrap in version 2021.6
+        wrap
+            maximum number of printed bases, old name is interleave_len
         """
+        if interleave_len != None:
```

Comment:
change to `if interleave_len is not None:`

Reply:
is there a difference between `!=` and `is not`?